### PR TITLE
Add Jonah greeting

### PR DESCRIPTION
**Pull request by Jonah-Northwood**
Creates simple method and call.

### CODE REVIEW

**Review by jquist24 — DISMISSED** [review]
Looks perfect.

**Review by lam00071 — DISMISSED** [review]
Greatttt! :D

**Review by jquist24 — DISMISSED** [review]
looks good.

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public String generateOutput() {
 
     builder.append(cassieSaysHello());
     builder.append(jeffSaysHello());
+    builder.append(jonahSaysHello());
```
Comment (lam00071) [inline src/main/java/hellos/Hellos.java:34]: Looks great to me! :D

**Review by lam00071 — DISMISSED** [review]
Let's finish this lol

**Review by NicMcPhee — DISMISSED** [review]

**Review by JacobJenness — APPROVED** [review]

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -88,11 +89,22 @@ private String blakeSaysHello() {
...
+    return "Cassie says 'Hello!'\n";
+  }
+  
+  private String jeffSaysHello() {
+    return "Jeff says 'Hello!'\n";
+  }
+  
+  private String jonahSaysHello() {
+    return "Jonah says 'Hi!'\n";
+
```
Comment (JacobJenness) [inline src/main/java/hellos/Hellos.java]:
```suggestion
}

```

**Review by JacobJenness — CHANGES_REQUESTED** [review]
I missed a single semicolon. Whoops

File: src/main/java/hellos/Hellos.java (hunk 2 above)
Comment (NicMcPhee) [reply src/main/java/hellos/Hellos.java]: Yup, this is why it isn't compiling.

**Review by schu4276 — APPROVED** [review]
Looks good!

**Review by JacobJenness — APPROVED** [review]

**Pull request merged**